### ISSUE TITLE
Numeric Check during Tuning 

### ISSUE THREAD

**Issue by JehandadKhan** [ROCm/MIOpen#513]

**Comment by atamazov** [comment]
Checks like `ConvForwardCheckNumerics()` or full verification of output?

**Comment by JehandadKhan** [comment]
@atamazov yeah, there has been instances of numeric issues during tuning. So this is a debug feature to help developers troubleshoot such problems.

**Comment by atamazov** [comment]
@JehandadKhan 
> there has been instances of numeric issues during tuning.

Perhaps _after_ tuning, i.e. tuned config fails to pass verification?

**Comment by atamazov** [comment]
The driver validates the final result of tuning, if `-V 1` is passed. Would you like to verify output after each tuning run?

**Comment by JehandadKhan** [comment]
> 
> 
> The driver validates the final result of tuning, if `-V 1` is passed. Would you like to verify output after each tuning run?

Yeah, that is the request. I think we would need to add the `ConvForwardCheckNumerics()` in the `generic_search.cpp` loop. Do you have a better approach in mind ?

**Comment by atamazov** [comment]
Let me clarify. AFAICS CheckNumerics doesn't perform verification for correctness. It checks for NaNs INFs and possibly other junk, I guess. Do we want this?

`ConvForwardCheckNumerics()` checks input _buffers_. I think we want to check _output_ buffer, right?

**Comment by ppanchad-amd** [comment]
@JehandadKhan @atamazov Is this take care of in the latest ROCm 6.0.2?

**Comment by atamazov** [comment]
@ppanchad-amd The feature is not implemented yet.

**Comment by JonathanLichtnerAMD** [comment]
Closing since MIOpen is being migrated to [ROCm/rocm-libraries](https://github.com/ROCm/rocm-libraries).  Please re-open there if you believe this is still an issue.
